### PR TITLE
changes in README.md to ease my frustration

### PR DESCRIPTION
**Pull request by Tadek1**
I am too green to make changes in the install process of _dotfiles_.  My original frustration was that after installing _dotfiles_, I had error displayed while starting vim:
`
Error detected while processing /home/tad/.vimrc:
line   65:
E185: Cannot find color scheme 'solarized'
Press ENTER or type command to continue
`
and colours where not displayed 'solarized'.  After long investigation (learning vim and git) I figured out the culprit:  plugins must be installed first.  

There was another problem with another plugin which required MAC (zerowidth/vim-copy-as-rtf?), but it is beyond the scope of this pull request.

Long story short: it takes longer then a week to learn vim.

### CODE REVIEW

**Comment by mscoutermarsh** [comment]
Hi @Tadek1!

Thanks! Appreciate it! Going to take a look at this soon + will add in some other changes to make it easier to understand.
